.github/actions/run-test/action.yml

```diff
@@ -36,6 +36,13 @@ runs:
       with:
         node-version: ${{ inputs.node-version }}
     - uses: ./.github/actions/enable-microphone-access
+    - name: Setup Ubuntu Binary Installation
+      if: ${{ runner.os == 'Linux' }}
+      run: |
+        if grep -q "Ubuntu 24" /etc/os-release; then
```

Comment:
Lets add a todo to remove it after https://github.com/electron/electron/issues/42510 has been fixed?

Also maybe do it only if command contains itest or etest?